Binary testing: "No configuration files"

Version: 2.0.0-rc.2

Example log on windows:

```
=== RUN   TestAccResourceID
cannot run Terraform provider tests: failed to create symlink at C:\Users\paul\AppData\Local\Temp\tftest-terraform-provider-random607016463\plugins-current035595298\terraform-provider-random to C:\Users\paul\AppData\Local\Temp\go-build128261702\b233\provider.test.exe: symlink C:\Users\paul\AppData\Local\Temp\go-build128261702\b233\provider.test.exe C:\Users\paul\AppData\Local\Temp\tftest-terraform-provider-random607016463\plugins-current035595298\terraform-provider-random: A required privilege is not held by the client.
FAIL    github.com/terraform-providers/terraform-provider-random/internal/provider      0.321s
```

Running as Administrator, in Developer Mode:

```
--- FAIL: TestAccResourceID (1.89s)
    testing_new.go:94: terraform failed: exit status 1
        
        stderr:
        
        Error: No configuration files
        
        Apply requires configuration to be present. Applying without a configuration
        would mark everything for destruction, which is normally not what is desired.
        If you would like to destroy everything, run 'terraform destroy' instead.
```

--- a/go.mod
+++ b/go.mod
@@ -25,8 +25,8 @@ require (
 	github.com/hashicorp/go-version v1.2.0
 	github.com/hashicorp/hcl/v2 v2.3.0
 	github.com/hashicorp/logutils v1.0.0
-	github.com/hashicorp/terraform-json v0.4.0
-	github.com/hashicorp/terraform-plugin-test v1.4.0
+	github.com/hashicorp/terraform-json v0.5.0
+	github.com/hashicorp/terraform-plugin-test v1.4.4
 	github.com/hashicorp/yamux v0.0.0-20181012175058-2f1d1f20f75d // indirect
 	github.com/keybase/go-crypto v0.0.0-20161004153544-93f5b35093ba
 	github.com/kylelemons/godebug v1.1.0 // indirect

